client/js/connection.js

```diff
@@ -2,7 +2,14 @@ let lastTimeout = 1000;
 let connection;
 let messageCallbacks = {};
 
-function startWebSocket() {
+//used by unit tests until jest supports mocking ESM static imports
+export function mockConnection() {
```

Comment:
All these exports/imports still work in the browser? In all relevant browsers?

Reply:
Not the imports. They are stripped as part of the minify code. Exports work: I've tested Chromium and Safari. 

Where is the list of supported browsers specified?

Reply:
There's no list but I generally just check [caniuse.com](https://www.caniuse.com/?search=export). It looks like it's not supported on the current Android Browser which should be a deal breaker (unless it simply doesn't understand it but everything still works).

Reply:
I don't have an android device. Per these [chrome-devtools "emulate & test other browsers"](https://developers.google.com/web/tools/chrome-devtools/device-mode/testing-other-browsers) instructions, it talks about downloading an Android emulator by way of the Android SDK. Which I guess that means, I can't just use F12 in Chromium to put the browser into an Android webkit emulation mode.

Does that sound right to you? If so, I'll take the time to install and try it out.

Reply:
I think we'll have enough Android devices in the community. I will test with mine when I have the time.